CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We added support for multiple languages for exporting to and importing references from MS Office. [#9699](https://github.com/JabRef/jabref/issues/9699)
 - We enabled scrolling in the groups list when dragging a group on another group. [#2869](https://github.com/JabRef/jabref/pull/2869)
 - We added the option to automatically download online files when a new entry is created from an existing ID (e.g., DOI). The option can be disabled in the preferences under "Import and Export". [#9756](https://github.com/JabRef/jabref/issues/9756)
+- We added Snuggletex for assisting with parsing our bib entries. [koppor#585](https://github.com/koppor/jabref/issues/585), [8712](https://github.com/JabRef/jabref/issues/8712)
```

Comment:
This is not an entry for a changelog.

You can do

```suggestion
- We added an integrity check for more special characters. [#8712](https://github.com/JabRef/jabref/issues/8712)
```

(The changelog for koppor#585 is already there)